KsApi/MockService.swift

```diff
@@ -66,7 +66,8 @@ internal struct MockService: ServiceType {
   fileprivate let fetchProjectStatsResponse: ProjectStatsEnvelope?
   fileprivate let fetchProjectStatsError: ErrorEnvelope?
 
-  fileprivate let fetchShippingRulesResponse: [ShippingRule]
+  fileprivate let fetchShippingRulesResponse: [ShippingRule]?
+  fileprivate let fetchShippingRulesError: ErrorEnvelope?
```

Comment:
i started doing something a lil new in [MockLiveStreamService](https://github.com/kickstarter/ios-oss/blob/master/LiveStream/Service/MockLiveStreamService.swift#L17) by bundling up the success/failure values into a single result. helps reduce the number ivars we have and makes it easier in the future to make sure we support the success/failure cases always

wanna give that a shot?